services/server/server.go

```diff
@@ -611,6 +618,49 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 	}, nil
 }
 
+// deleteAllAlertRules deletes all alerts present rules.
+// Returns nil in case of success, otherwise an error.
+func (s *Server) deleteAllAlertRules() error {
+	alertRuleIds := s.getAlertRuleIds()
+
+	if len(alertRuleIds) > 0 {
+		err := s.db.InTransaction(func(tx *reform.TX) error {
+			return models.RemoveRules(tx.Querier, alertRuleIds)
+		})
+		if err != nil {
+			return err
+		}
+
+		s.vmalert.RequestConfigurationUpdate()
+	}
+
+	return nil
+}
+
+// getAlertRuleIds lists the alert rules ids into an array.
+// Error is not returned, when occurres an empty array is returned.
+func (s *Server) getAlertRuleIds() []string {
+	var rules []*models.Rule
+	err := s.db.InTransaction(func(tx *reform.TX) error {
+		var err error
+		rules, err = models.FindRules(tx.Querier)
+
+		return err
+	})
+
+	if err == nil {
+		ruleIds := make([]string, len(rules))
```

Comment:
minor: in Go it's common to have initialisms or acronyms consistent case, so `ruleIDs` instead of `ruleIds`. The same for the `alertRuleIds` above.
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I though that ID is for Identification Document where Id just an abbreviation of Identification.
I got you. Ids could also be understood as Idstein where they used to burn witches.
So I'll change it.

Reply:
Does not exist anymore.